lib/HLSL/DxilValidation.cpp

```diff
@@ -2156,6 +2174,15 @@ static void ValidateDxilOperationCallInProfile(CallInst *CI,
   // Is called from a library function
   bool isLibFunc = shaderKind == DXIL::ShaderKind::Library;
 
+  // get the node launch type
+  DxilModule &DM = ValCtx.DxilMod;
+  DXIL::NodeLaunchType NLT = DXIL::NodeLaunchType::Invalid;
+
+  if (DM.HasDxilEntryProps(F)) {
+    DxilEntryProps &entryProps = DM.GetDxilEntryProps(F);
```

Comment:
This can be merged with earlier code that gets the properties under this branch:
https://github.com/microsoft/DirectXShaderCompiler/pull/6108/files#diff-3d74ec29820bb5a94491fb4d46c80d37c89b0e72c55df5394fea58236c1f5300R2163

Reply:
I think the link may be broken?

Reply:
It links to above code in this file, which is collapsed by default.  Just look around line 2163 for `if (ValCtx.DxilMod.HasDxilFunctionProps(F))`.  That branch can be merged with this.